CONFIG.md

```diff
@@ -51,3 +51,4 @@ This list will be updated as properties are migrated:
 | org.jitsi.videobridge.ONSTAGE_PREFERRED_FRAME_RATE | videobridge.cc.onstage-preferred-framerate | |
 | org.jitsi.videobridge.ENABLE_ONSTAGE_VIDEO_SUSPEND | videobridge.cc.enable-onstage-video-suspend | |
 | org.jitsi.videobridge.TRUST_BWE | videobridge.cc.trust-bwe | |
+| org.jitsi.videobridge.PADDING_PERIOD_MS | videobridge.cc.padding-period | |
```

Comment:
This should probably have the comment that this is a duration, like the other properties have.

Reply:
yup, missed that.